go.mod

```
@@ -1,6 +1,6 @@
 module github.com/migtools/udistribution
 
-go 1.18
+go 1.22.7
```

Please do not bump go like this. Why are we pinning to 1.22.7?

I'm trying to avoid this repo from causing https://github.com/sigstore/fulcio/issues/1899

From my checking this can just be go 1.21
```suggestion
go 1.21
```

---

@kaovilai Should we make similar change to the https://github.com/openshift/openshift-velero-plugin/pull/290/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R3 ?

go there was bumped to a version that is aligned with other plugins and this was the 1.22.7 so I thought it's correct to use.

---

This repo specifically is not being built directly. I want the go directive to be as relaxed as possible.

In openshift-velero-plugin go directive should not move if not necessary as well. Golang CLI version alignment is still desired and should be achieved via Dockerfile only.